demo/summer-2016.html

```diff
@@ -209,6 +211,10 @@
 \end{document}</pre>
 
 <script src="kitchen-sink/require.js"></script>
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```

Comment:
just FYI: it is okay for the demo but you will not want it in production (because if anything goes wrong with the hotlinked file on server which is not under your control then you're in troubles)